src/main/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerNodeSelectionListener.java

```diff
@@ -14,18 +14,20 @@
 	@Override
 	public void handleEvent(RowsSetEvent e) {
 
-		if (e.containsColumn(CyNetwork.SELECTED) && e.getSource() == 
-				PathLinkerPanel._applicationManager.getCurrentNetworkView().getModel().getDefaultNodeTable()) {
+		if (e.getSource() != PathLinkerControlPanel._applicationManager.getCurrentNetworkView().getModel().getDefaultNodeTable())
```

Comment:
Can you add a comment here describing what these if statements are doing? I think this one is to check if the source of the event is not the current network's node table.